backend/shared/shared/azure_adfs/tests/test_adfs_login.py

```diff
@@ -27,7 +27,9 @@ def test_get_member_objects(requests_mock):
     auth_backend = HelsinkiAdfsAuthCodeBackend()
 
     member_objects_response = {
-        "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#Collection(Edm.String)",
+        "@odata.context": (
+            "https://graph.microsoft.com/v1.0/$metadata#Collection(Edm.String)"
+        ),
```

Comment:
Why all the formatting changes in this file i.e. putting strings into parentheses? Is black doing this or something else?

Reply:
Yes, new black version formats these also. The line was over 88 lines and in `dict` you need to put value in parentheses if it's on the different line that key.